crypto/src/main/java/net/corda/v5/crypto/CordaOID.java

```diff
@@ -0,0 +1,37 @@
+package net.corda.v5.crypto;
+
+/**
+ * OIDs used for the Corda platform. All entries MUST be defined in this file only and they MUST NOT be removed.
+ * If an OID is incorrectly assigned, it should be marked deprecated and NEVER be reused again.
+ */
+
+public final class CordaOID {
+    private CordaOID() {
+        // this class is never constructed; it exists for the static strings only
+    }
+    
+    /** Assigned to R3, see http://www.oid-info.com/cgi-bin/display?oid=1.3.6.1.4.1.50530&action=display */
+    public static final String OID_R3_ROOT = "1.3.6.1.4.1.50530";
+
+    /** OIDs issued for the Corda platform. */
+    public static final String OID_CORDA_PLATFORM = "1.3.6.1.4.1.50530.1";
```

Comment:
This used to use `OID_R3_ROOT`. Should we OID_R3_ROOT + ".1" instead?

Reply:
done